src/frontends/common_translators/src/op/angle.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_translators.hpp"
+#include "openvino/frontend/complex_type_mark.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/less.hpp"
+#include "openvino/op/select.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace common_translators {
+
+using namespace ov::op;
+using namespace ov::frontend;
+using namespace std;
+using namespace ov;
```

Comment:
`using namespace ov;` is doing nothing if namespace ov is open.
`using namespace ov::frontend;` I think is also not needed as I do not see any call that may be using it.

Similar comments for other files

Reply:
Great catch, removed